ipn/ipnstate/ipnstate.go

```diff
@@ -145,6 +145,9 @@ type PeerStatus struct {
 	PeerAPIURL   []string
 	Capabilities []string `json:",omitempty"`
 
+	// SSH_HostKeys are the node's SSH host keys, if known.
+	SSH_HostKeys []string `json:"sshHostKeys,omitempty"`
```

Comment:
views.Slice[string]?

Reply:
I'd rather do them all at the same time. For now I value consistency.

Reply:
we kinda already are inconsistent here, but sure.

Reply:
Also, you'd want the pointer. Above we do have one `Tags *views.Slice[string]`

Reply:
e.g. Tags and PrimaryRoutes

Reply:
I really wish we didn't need that pointer. Ugh.

/cc @dsnet (for omitzero in json)

Reply:
yeah you'd need the pointer for `omitempty` to work